chapters/arrays.tex

```diff
@@ -1268,10 +1268,20 @@ \subsection{Element-wise Division}\label{array-element-wise-division}\label{elem
 
 \subsection{Element-wise Exponentiation}\label{element-wise-exponentiation}
 
-Exponentiation \lstinline!a ^ b! is defined as \lstinline[language=C]!pow(double a, double b)! in the ANSI~C library if both \lstinline!a! and \lstinline!b! are
-\lstinline!Real! scalars. A \lstinline!Real! scalar value is returned.  If \lstinline!a! or \lstinline!b! are \lstinline!Integer! scalars, they are
-automatically promoted to \lstinline!Real!.  Consequences of exceptional situations, such as ($\text{\lstinline!a!} = 0.0$ and $\text{\lstinline!b!} \leq 0.0$,
-$\text{\lstinline!a!} < 0$ and \lstinline!b! is not an integer) or overflow are undefined.
+Exponentiation \lstinline!a ^ b! always returns a \lstinline!Real! scalar value, and it is required that \lstinline!a! and \lstinline!b! are scalar \lstinline!Real! or \lstinline!Integer! values.
+The result should correspond to mathematical exponentiation (ideally the correctly rounded result based on the inputs) with the following special cases:
```

Comment:
Unless _based on the inputs_ is trying to say something that I don't see, it is probably better to leave it out:
```suggestion
The result should correspond to mathematical exponentiation (ideally the correctly rounded result) with the following special cases:
```

Reply:
Actually, since we don't speak of correctly rounded anywhere else in the specification, we probably shouldn't do it here either:
```suggestion
The result should correspond to mathematical exponentiation with the following special cases:
```